src/chainbin/nnet3-chain-copy-egs.cc

```diff
@@ -115,13 +318,16 @@ int main(int argc, char *argv[]) {
         }
       } else if (count > 0) {
         NnetChainExample eg = example_reader.Value();
+        NnetChainExample eg_out;
+        if (left_context != -1 || right_context != -1)
+          Modify_chain_context(eg, left_context, right_context, &eg_out);
         if (frame_shift != 0)
```

Comment:
You have to modify the logic a little here because eg_out will be empty.
e.g Add `else eg_out = eg;`